nucliadb/tests/standalone/fixtures.py

```diff
@@ -28,7 +28,7 @@
 async def knowledgebox_one(nucliadb_manager):
     kbslug = str(uuid.uuid4())
     data = {"slug": kbslug}
-    resp = await nucliadb_manager.post(f"/{KBS_PREFIX}", json=data)
+    resp = await nucliadb_manager.post(f"/{KBS_PREFIX}", json=data, timeout=None)
```

Comment:
Random change?

Reply:
oh yes, this helps debugging. When you put a breakpoint, if timeout is not none eventually the request handling task is cancelled and you get kicked out